stl/inc/xutility

```diff
@@ -5219,6 +5252,7 @@ _CONSTEXPR20 _FwdIt rotate(_FwdIt _First, _FwdIt _Mid, _FwdIt _Last) {
     // exchange the ranges [_First, _Mid) and [_Mid, _Last)
     // that is, rotates [_First, _Last) left by distance(_First, _Mid) positions
     // returns the iterator pointing at *_First's new home
+    _REQUIRE_CPP17_MUTABLE_ITERATOR(_FwdIt);

```

Comment:
As we discussed in Discord, the `_REQUIRE_MEOW` checks were intended for parallel algorithms only and should not be applied to classic non-parallel algorithms (as they may break squirrely user code; we have enough trouble when we add optimizations that dispatch on iterator strength).

You removed several occurrences but `rotate(first, mid, last)` here still needs to be fixed. (I am unsure if this is the last one.)